src/Quartz/DateBuilder.cs

```diff
@@ -259,7 +247,7 @@ public static DateTimeOffset TomorrowAt(int hour, int minute, int second)
         ValidateMinute(minute);
         ValidateHour(hour);
 
-        DateTimeOffset now = DateTimeOffset.Now;
+        DateTimeOffset now = TimeProvider.System.GetLocalNow();
```

Comment:
maybe needs parameter `TimeProvider? timeProvider = null` and then we default to system?

Reply:
Adjusted in next PR, let me know if suitable.